operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1114,6 +1114,15 @@ ruler:
     annotations: {}
     labels: {}
 
+  # -- Dedicated service account for ruler pods.
+  # If not set, the default service account defined at the begining of this file will be used.
+  # This service account can be used even if the default one is not set.
+  serviceAccount:
+    create: false
+    name: ""
```

Comment:
can you add a doc comment (`# --`) about the meaning of the empty string?

Reply:
Done :)